x-pack/plugins/lens/public/datasources/form_based/dimension_panel/dimension_panel.test.tsx

```diff
@@ -351,15 +351,15 @@ describe('FormBasedDimensionEditor', () => {
     expect(options).toHaveLength(3);
 
     expect(options![0].label).toEqual('Records');
-    expect(options![1].options!.map(({ label }) => label)).toEqual([
+    expect(options![1].options!.map(({ label }: { label: string }) => label)).toEqual([
```

Comment:
the repeated `{ label: string }` type definitions here are incredibly un-DRY. Rather than statically re-defining each arg, you should re-use EUI's combo box types instead:
```suggestion
    expect(options![1].options!.map(({ label }: EuiComboBoxOptionOption) => label)).toEqual([
```

Reply:
@1Copenut there are two other files in this PR/diff that are still setting manual `{ label: string }` types

Reply:
👍 I've updated them and will be pushing in just a moment.